spec/ParseObject.spec.js

```diff
@@ -1980,4 +1980,31 @@ describe('Parse.Object testing', () => {
       done();
     })
   });
+
+  it ('Update object field should store exactly same sent object', async (done) => {
```

Comment:
Using async/await, there’s no need for done as a param and call at the end of the method